src/lib/collab/stack-client.js

```diff
@@ -383,5 +383,13 @@ export class ServiceClient {
       telepointerDoc
     )
   }
+
+  postImage = async (name, noteId, file) =>
+    this.stackClient.fetchJSON(
+      'POST',
+      this.path(noteId, `images?Name=${name}`),
+      file,
+      { headers: { 'Content-Type': 'image/jpeg' } }
```

Comment:
What if the image is a GIF,  PNG or binary? 🤔

Reply:
it has to be a mime type image but I'll update the image type dynamically